src/ray/util/logging_test.cc

```diff
@@ -122,6 +123,27 @@ TEST(PrintLogTest, CallstackTraceTest) {
   EXPECT_TRUE(ret2.find("TestFunctionLevel2") != std::string::npos);
 }
 
+/// Catch abort signal handler for testing RAY_CHECK.
+/// We'd better to run the following test case manually since process
+/// will terminated if abort signal raising.
+/*
+bool get_abort_signal = false;
+void signal_handler(int signum) {
+  RAY_LOG(WARNING) << "Interrupt signal (" << signum << ") received.";
+  get_abort_signal = signum == SIGABRT;
+  exit(0);
+}
+
+TEST(PrintLogTest, RayCheckAbortTest) {
```

Comment:
How did it work before?

Reply:
I add it only for testing manually because process will abort and exit if signal catched.